### PR TITLE
Add KSirK

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/games/ksirk/-/issues/1
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/games/ksirk/-/merge_requests/6

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.ksirk

cc https://github.com/orgs/flathub/teams/kde

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by CarlSchwan** [comment]
Looks good to me and I merged https://invent.kde.org/games/ksirk/-/merge_requests/6

File: appdata.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
+From fe6d6ac54134ed80be121952893bf4aaa29995a8 Mon Sep 17 00:00:00 2001
```
Comment (CarlSchwan) [inline appdata.patch:1]: Maybe merge the two patches? Probably not that important since the patch won't be required in the future

Comment (Mailaender) [reply appdata.patch:1]: I'd like to keep the commit log readable.

**Pull request closed without merging**